src/group_theory/subgroup/basic.lean

```diff
@@ -2123,6 +2123,13 @@ end, by rintro ⟨y, hy, z, hz, rfl⟩; exact mul_mem_sup hy hz⟩
 lemma mem_sup' : x ∈ s ⊔ t ↔ ∃ (y : s) (z : t), (y:C) * z = x :=
 mem_sup.trans $ by simp only [set_like.exists, coe_mk]
 
+@[to_additive]
+lemma mem_closure_pair {x y z : C} : z ∈ closure ({x, y} : set C) ↔ ∃ m n : ℤ, x ^ m * y ^ n = z :=
```

Comment:
Seems like there might be a lint error involving this lemma?

Reply:
Maybe merging master would be a good idea?

Reply:
Actually I think it's the mem_closure_pair found in group_theory/submonoid/membership